### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#418]
# Related Addresses

```
0x0c34fc64550fcf174e7ef848515cfdc961971168
0x0dae17d9884d59fe096970cee442a7707e6892a4
0x2b2696759d6d77674fa2d17a176b2b650ef29c56
0x50a6606d425815dcd96d3727218ccb939abe7e4c
0x6f23df8afc81bcb1edc25d1ff5052e566ac34580
0x85accb9e1c0483ae84734140b414d1102ee22cd4
0xa5d0040c847ca583076962693ebaac191fd827f5
0xba5954cd1caf1f3a7d6b69f8d3ae7df1d5ae5792
0xdaa9a26de24b063049d5869a251b46cebdeb8242
0xe039e1d8194a335bbe1c21ab6bd5698289d05bb3
0x2368ed7c29e8d05be392b5b9bfc81d2e30bc0d33
```

All addresses are originally funded by address 0x3631a6b1318a73dd2dbf890713a8aaa2c98c9a50 on Nov 19, 2021. They all then proceed to do a swap using Metamask Router on Polygon Chain.

# Methodology

I used Alchemy's API to see which address originally funded all 30k eligible addresses on Polygon Mainnet, then used excel to check if multiple wallets were funded by the same EOA address. I then checked their transactions on Polygonscan to try and find similarities.

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by zlgitol** [comment]
0x3631a6b1318a73dd2dbf890713a8aaa2c98c9a50 is whitelist. Are these valid? @shanefontaine

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy . We have verified that the addresses in this report are Sybil attackers.

The report included 11 eligible addresses as Sybil attackers which means you are eligible for 1534.960110583613630641 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).


The qualified addresses are as follows:

```
0x0c34fc64550fcf174e7ef848515cfdc961971168
0x0dae17d9884d59fe096970cee442a7707e6892a4
0x2b2696759d6d77674fa2d17a176b2b650ef29c56
0x50a6606d425815dcd96d3727218ccb939abe7e4c
0x6f23df8afc81bcb1edc25d1ff5052e566ac34580
0x85accb9e1c0483ae84734140b414d1102ee22cd4
0xa5d0040c847ca583076962693ebaac191fd827f5
0xba5954cd1caf1f3a7d6b69f8d3ae7df1d5ae5792
0xdaa9a26de24b063049d5869a251b46cebdeb8242
0xe039e1d8194a335bbe1c21ab6bd5698289d05bb3
0x2368ed7c29e8d05be392b5b9bfc81d2e30bc0d33
```

**Comment by shanefontaine** [comment]
In addition to the data you provided, it looks like:

* All addresses have near identical transaction history on Polygon
* All addresses have near identical Hop protocol behavior

address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0x2368ed7c29e8d05be392b5b9bfc81d2e30bc0d33 |   | 2 | 0 | 1 | 1 | 0 | 0 | 4596.189764
0x0c34fc64550fcf174e7ef848515cfdc961971168 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1906.571422
0x0dae17d9884d59fe096970cee442a7707e6892a4 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1891.421913
0x2b2696759d6d77674fa2d17a176b2b650ef29c56 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1862.31615
0x85accb9e1c0483ae84734140b414d1102ee22cd4 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1818.03096
0x50a6606d425815dcd96d3727218ccb939abe7e4c |   | 2 | 0 | 1 | 1 | 0 | 0 | 1782.737601
0xba5954cd1caf1f3a7d6b69f8d3ae7df1d5ae5792 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1759.159515
0x6f23df8afc81bcb1edc25d1ff5052e566ac34580 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1638.263298
0xdaa9a26de24b063049d5869a251b46cebdeb8242 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1603.392045
0xe039e1d8194a335bbe1c21ab6bd5698289d05bb3 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1311.378647
0xa5d0040c847ca583076962693ebaac191fd827f5 |   | 2 | 0 | 1 | 1 | 0 | 0 | 1107.059778

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
